web/tree/util.js

```diff
@@ -0,0 +1,86 @@
+window.history.replaceState(null, document.title, window.location.href.replace(/\/?$/, '/'));
+
+function addNewPost(spur){
+  var title = document.getElementsByTagName("input")[0].value;
+  var comment = document.getElementsByTagName("textarea")[0].value;
+  
+  var data = {
+    pax: "/",
+    sup: spur,
+    hed: title,
+    txt: comment,
+  };
+
+  var params = {
+    appl: "fora",
+    mark: "fora-post",
+  };
+
+  window.urb.send(data, params, function(error, result){
+    window.location.href = "../../fora.last-post?by=~"+window.urb.ship;
+  });
+  
+
+
+}
+
+
+function addNewComment(spur){
+  var comment = document.getElementsByTagName("textarea")[0].value;
+
+  var data = {
+    pax: "/",
+    sup: spur,
+    txt: comment,
+  };
+
+  var params = {
+    appl: "fora",
+    mark: "fora-comment",
+  };
+
+  window.urb.send(data, params, function(error, result){
+    window.location.reload(true);
+  });
+}
+
+function editPlanInfo(viewmode){
+var edit = document.getElementsByClassName('edit')[0];
+
+  who = document.getElementsByTagName('h6')[0];
+  loc = document.getElementById('loc');
+  if (viewmode) {
+    who.style.display = 'block';
+
+    var whoIn = document.createElement("input");
+    whoIn.placeholder = who.innerHTML || 'Sun Tzu';
```

Comment:
Should this still be in there? Looks a bit like a debugging artifact.

Reply:
not sure exactly what you're talking about here but if you mean the 'Sun Tzu' thing, thats just to keep it the same as the old home page when you edit your name.

Reply:
Ok, that's fine then.